functions/src/sumologic/sumologic.ts

```diff
@@ -25,6 +25,12 @@ export const logSmsSent = () => {
     })
 }
 
+export const logCallRating = (rating: number) => {
+    sendLog({
+        callRating: rating,
```

Comment:
need additional data: where the user using daily or twilio

Reply:
what about adding also the `roomId`, in case we want to aggregate the data with:
- call duration
- number of participants
- network statuses (if possible)

in order to better understand what could have led to bad ratings ?